src/main/java/edu/harvard/iq/dataverse/util/json/JSONLDUtil.java

```diff
@@ -192,13 +202,25 @@ public static DatasetVersion updateDatasetVersionMDFromJsonLD(DatasetVersion dsv
                                         "Cannot specify " + JsonLDTerm.schemaOrg("license").getUrl() + " and "
                                                 + JsonLDTerm.termsOfUse.getUrl());
                             }
-                            setSemTerm(terms, key, TermsOfUseAndAccess.defaultLicense);
+                            try {
+                                if (dsv.getTermsOfUseAndAccess() == null && !dsv.getDataset().isReleased()) {

```

Comment:
why only if the dataset hasn't been released? A new version could have a different license?